packages/manager/apps/dedicated/client/app/billing/autoRenew/resiliation/resiliation.routing.js

```diff
@@ -0,0 +1,38 @@
+import { BillingService as Service } from '@ovh-ux/manager-models';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.account.billing.autorenew.resiliation', {
+    url: '/resiliation?serviceId&serviceName',
+    component: 'ovhManagerBillingResiliation',
+    translations: { value: ['.'], format: 'json' },
```

Comment:
Maybe we could use an annotation instead
```suggestion
.run(/* @ngTranslationsInject:json ./translations */
```